.eslintrc.yml

```diff
@@ -0,0 +1,2 @@
+env:
```

Comment:
this needs to be merged with the same name file in the api-bench root - otherwise it won't work :)

Reply:
so this means I need to merge the eslintrc.yml from the the root repo into this yml?

Sorry if that is a stupid question, I am new to this kind of workflow. 😅

The eslintrc is the setup for the Typescript warnings and error sytem am I right?

Reply:
> so this means I need to merge the eslintrc.yml from the the root repo into this yml?

Yes

> Sorry if that is a stupid question, I am new to this kind of workflow. 😅

No worries :)

> The eslintrc is the setup for the Typescript warnings and error sytem am I right?

Not quite, eslint is something on top, that helps detect issues with javascript and typescript.

Reply:
I've made another commit, merging the two .yml files